src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -1600,6 +1600,90 @@ public async Task ValidateLocationHeaderFieldForPostRequests(EntitySourceType en
             }
         }
 
+        /// <summary>
+        /// Test to validate that when the property rest.request-body-strict is absent from the rest runtime section in config file, DAB runs in strict mode.
+        /// In strict mode, presence of extra fields in the request body is not permitted and leads to HTTP 400 - BadRequest error.
+        /// </summary>
+        /// <param name="includeExtraneousFieldInRequestBody">Boolean value indicating whether or not to include extraneous field in request body.</param>
+        [DataTestMethod]
+        [TestCategory(TestCategory.MSSQL)]
+        [DataRow(false, DisplayName = "Mutation operation passes when no extraneous field is included in request body and rest.request-body-strict is omitted from the rest runtime section in the config file.")]
+        [DataRow(true, DisplayName = "Mutation operation fails when an extraneous field is included in request body and rest.request-body-strict is omitted from the rest runtime section in the config file.")]
+        public async Task ValidateStrictModeAsDefaultForRestRequestBody(bool includeExtraneousFieldInRequestBody)
+        {
+            string entityJson = @"
+            {
+                ""entities"": {
+                    ""Book"": {
+                        ""source"": {
+                        ""object"": ""books"",
+                        ""type"": ""table""
+                        },
+                        ""permissions"": [
+                        {
+                            ""role"": ""anonymous"",
+                            ""actions"": [
+                            {
+                                ""action"": ""*""
+                            }
+                            ]
+                        }
+                        ]
+                    }
+                }
+            }";
+
+            string configJson = TestHelper.AddPropertiesToJson(TestHelper.BASE_CONFIG, entityJson);
```

Comment:
Should comment either here or in test description: `TestHelper.BASE_CONFIG` omits the `rest.request-body-strict` option. It is not obvious when looking at this test in isolation.

Reply:
done